sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -215,6 +215,7 @@ statement
     | SET .*?                                                          #setConfiguration
     | RESET                                                            #resetConfiguration
     | DELETE FROM multipartIdentifier tableAlias whereClause           #deleteFromTable
```

Comment:
Also, we should make whereClause too for DELETE.

Reply:
Will open another pr for this.